astropy/modeling/functional_models.py

```diff
@@ -1469,6 +1551,53 @@ def bounding_box(self):
         return ((self.y_0 - dy, self.y_0 + dy),
                 (self.x_0 - dx, self.x_0 + dx))
 
+    def integral(self, axis='x'):
```

Comment:
Don't know that I like the 2D implementation, as it implicitly assumes the other coordinate equals 0.